src/Util/ResourceClassInfoTrait.php

```diff
@@ -34,7 +34,7 @@ trait ResourceClassInfoTrait
     /**
      * @var ResourceMetadataFactoryInterface|null
      */
-    private $resourceMetadataFactory;
+    protected $resourceMetadataFactory;
```

Comment:
why?

Reply:
Because of property conflicts with trait composition: sometimes the property was `protected`, sometimes `private`. PHP doesn't allow that and does not provide a way to resolve conflicts like it does for methods.